app/components/pop-in-campaigns.js

```diff
@@ -0,0 +1,24 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { htmlSafe } from '@ember/string';
+
+export default Component.extend({
+  classNames: ['pop-in-campaigns'],
+  classNameBindings: ['isClose:collapse'],
+  title: computed('isClose', function() {
+    if(this.isClose) {
+      return "code parcours"
+    }
+    return htmlSafe('Vous avez un <span class="bold">code parcours ?</span>');
+  }),
+  isClose: null,
+
+  actions: {
+    closePopIn() {
+      this.set('isClose', true);
```

Comment:
Tu peux avoir une seule action ici et utiliser `this.toggleProperty`
(https://api.emberjs.com/ember/3.11/classes/EmberObject/methods/toggleProperty?anchor=toggleProperty)

Reply:
Je ne connaissais pas merci !